_episodes/03-inventory.md

```diff
@@ -18,7 +18,7 @@ In 2017, OTN deployed the [SnipeIT Inventory](https://ops.oceantrack.org/snipeit
 
 ## General Tasks
 
-OTN Field technicians utilise SnipeIT on a regular basis, mainly doing the following activities:
+OTN Field technicians utilise SnipeIT on a regular basis, mainly doing the following activities.
```

Comment:
I actually think this should still be a colon

Reply:
i agree